docs/awesome-pets/papers/applications/ppml/fl/application.md

```diff
@@ -157,6 +159,8 @@ Keywords: Split Learning + Sparsification
 
 - [Exploiting Unlabeled Data in Smart Cities using Federated Learning](https://arxiv.org/pdf/2001.04030.pdf)
 
+- [A Mobility Forecasting Framework with Vertical Federated Learning](https://ieeexplore.ieee.org/abstract/document/9842629)
+
```

Comment:
It is not recommended to include C-level conferences in the awesome-list

Reply:
Thank you for your suggestion.